docs/book/helper/form-month-select.md

```diff
@@ -21,3 +21,30 @@ echo $this->formMonthSelect($monthYear);
 // <select name="monthyear[month]"> ... </select>
 // <select name="monthyear[year]"> ... </select>
 ```
+
+## Advanced usage
+
+### Render each element individually
+
+This can be archived by passing a custom pattern via `setPattern()`:
+
+```php
+<div class="row">
+    <div class="col-lg-7">
+        <!-- Render the select field for month only -->
+        <?=
+            $this->formMonthSelect()
+            ->setPattern('MMMM')
+            ->render($element);
+        ?>
+    </div>
+    <div class="col-lg-5">
+        <!-- Render the select field for day only -->
+        <?=
+             $this->formMonthSelect()
+            ->setPattern('y')
+            ->render($element);
+        ?>
```

Comment:
I think, the real goal is a different one: you want to render the elements separately and not to set the pattern manually.

Therefore I suggest a solution which adds two new methods to allow the rendering of the month and year elements separately. This protects the internal processing of the pattern.

The underlying `FormSelect` helper allows already the rendering of parts:

https://github.com/laminas/laminas-form/blob/1f87e9f98874c7f82b8bf2f9b85c902c8d2dd04c/src/View/Helper/FormSelect.php#L177-L181
https://github.com/laminas/laminas-form/blob/1f87e9f98874c7f82b8bf2f9b85c902c8d2dd04c/src/View/Helper/FormSelect.php#L253-L255

The `FormCollection` helper has a method to render the template:

https://github.com/laminas/laminas-form/blob/1f87e9f98874c7f82b8bf2f9b85c902c8d2dd04c/src/View/Helper/FormCollection.php#L166-L172

Reply:
We could instead define a prefix, separator and suffix property (with getters and setters) which get added to the internal array which holds the elements which should be rendered.

Reply:
Why so complicated? What's wrong with my proposal?

> I suggest a solution which adds two new methods to allow the rendering of the month and year elements separately.